test/ContentEncodingHelperTest.cpp

```diff
@@ -23,6 +23,27 @@ TEST_P(ContentEncodingHelperFixture, HeaderIsNotSetCorrectly) {
   ASSERT_EQ(header[http::field::content_encoding], headerValue);
 }
 
+TEST(ContentEncodingHelper, TestHeadersAreInsertedCorrectly) {
+  http::header<false, http::fields> header;
+  setContentEncodingHeaderForCompressionMethod(CompressionMethod::GZIP, header);
+  setContentEncodingHeaderForCompressionMethod(CompressionMethod::DEFLATE,
+                                               header);
+  setContentEncodingHeaderForCompressionMethod(CompressionMethod::DEFLATE,
+                                               header);
+
+  auto [current, end] = header.equal_range(http::field::content_encoding);

```

Comment:
Can also be `std::ranges::equal` with a vector of the three strings.

Reply:
Doesn't work actually because equal_range seems to return a special type of iterator